xml/app_scf_values_yaml.xml

```diff
@@ -17,10 +17,11 @@
   the current &productname; 1.3 release.
  </para>
 
-<screen>apiVersion: 2.14.5+cf2.7.0.0.g6360c016
+<screen>apiVersion: 2.15.2+cf3.6.0.0.gde1bd02f
+appVersion: 1.13.1
```

Comment:
This is actually a typo (that has since been fixed in the live version): `appVersion` should be `1.3.1`.

Reply:
:heavy_check_mark: